File: ts/project/.sdk/tm/csharp/test/ExistsTest.cs
// ProjectName SDK exists test.

using Xunit;

using ProjectNameSdk;

namespace ProjectNameSdk.Test;

public class ExistsTest
{
    [Fact]
    public void TestMode()
    {
        var testsdk = ProjectNameSDK.TestSDK(null, null);
        Assert.NotNull(testsdk);
    }
}

File: ts/project/.sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { ProjectNameSDK } from '..'


describe('exists', async () => {

  test('test-mode', () => {
    const testsdk = ProjectNameSDK.test()
    equal(testsdk instanceof ProjectNameSDK, true,
      'ProjectNameSDK.test() must return a client synchronously')
  })

})
